.gitmodules

```diff
@@ -9,3 +9,6 @@
 	path = core/packages/contracts/lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
 	branch = v4.8.2
+[submodule "cumulus"]
+	path = cumulus
+	url = https://github.com/Snowfork/cumulus
```

Comment:
@alistair-singh 
In this PR I change the cumulus integration as submodule here and then in this [commit](https://github.com/Snowfork/cumulus/commit/4ae67639c2e1ebe1e048432ab71ed074bdc0530d) make snowbridge as local dependency. 
IMHO could make more sense for dev workflow when we're doing heavily refactoring at this moment. Please let me know what you think.

Reply:
I am not opposed to this change. I am using vscode to develop cumulus and snowbridge by having them in the same vscode workspace. So this change will not affect me very much. Let's see what the rest of the team says.